mkosi.functions

```diff
@@ -0,0 +1,34 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+make_sysext_unsigned() {
+    if ! ((SYSEXT)); then
+        return
+    fi
+
+    mkdir -p /usr/lib/systemd/repart/definitions/sysext-unsigned.repart.d
+    cat >/usr/lib/systemd/repart/definitions/sysext-unsigned.repart.d/10-root.conf <<EOF
+[Partition]
+Type=root
+Format=erofs
+CopyFiles=/usr/
+Minimize=best
+EOF
+
+    mkdir -p "$1"/usr/lib/extension-release.d
+    cp /usr/lib/os-release "$1"/usr/lib/extension-release.d/extension-release.systemd
+    cat >>"$1"/usr/lib/extension-release.d/extension-release.systemd <<EOF
+SYSEXT_ID=systemd
```

Comment:
this also needs ID= matching the image it is being built for

Reply:
@bluca See the line just above this

Reply:
ah it's copied verbatim - mmh we usually don't want to copy the full thing, maybe grep ID and (if present) VERSION_ID or SYSEXT_LEVEL from it instead? That's how we usually build these in the CI